Say Their Names/Source/Controller/Donations/DonationsViewCode.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  DonationsViewCode.swift
+//  Say Their Names
+//
+//  Created by Hakeem King on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+/// The UI for Donations
+final class DonationsViewCode: UIView {
+    
+    private lazy var donationsLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Donations"
+        label.textAlignment = .center
+        
+        return label
+    }()
+    
+    init() {
+        super.init(frame: .zero)
```

Comment:
```suggestion
    init(frame: CGRect) {
        super.init(frame: frame)
```

if you do it like that, still can call `init()` but also `init(frame: x)`